### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Zemurin**
- regions
- relations
- religions

### CODE REVIEW

File: EU4ToVic2Tests/MapperTests/CultureMapperTests.cpp [hunk 1]
```diff
@@ -19,7 +19,7 @@ CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
 TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.*/
 
-
+/*
```
Comment (Idhrendur) [inline EU4ToVic2Tests/MapperTests/CultureMapperTests.cpp:22]: Is there a reason these are all commented out? If so, please create a todo tied to a github issue to deal with that reason. If not, please restore them.

**Review by Idhrendur — APPROVED** [review]
It looks like the usual light changes. Anything I should have looked at more closely?

File: EU4ToVic2Tests/MapperTests/CultureMapperTests.cpp (hunk 1 above)
Comment (Zemurin) [reply EU4ToVic2Tests/MapperTests/CultureMapperTests.cpp:22]: they rely on a mock I deleted when cleaning regions. mappers are next on my list so I'll be revising this in the next PR.

**Comment by Zemurin** [comment]
No logic changes if that's what you're asking. I dropped entire religion block that was unused and cleaned up a lot of minor details, but it's mostly cosmetics and tests.

**Pull request merged**